app/decorators/table_sorters/hearings_sorter.rb

```diff
@@ -0,0 +1,21 @@
+# frozen_string_literal: true
+
+module TableSorters
+  class HearingsSorter
+    def initialize(hearings, sort_order)
+      @hearings = hearings
+      @sort_order = sort_order
+    end
+
+    def sorted_hearing(hearing)
+      hearing.hearing_days.map(&:to_datetime).sort
+    end
+
+    private
```

Comment:
should probably be protected rather than private

Reply:
👍 I have changed this to protected